### ISSUE TITLE
Question: how to overwrite or remove quoted keys (like the "helm.sh/resource-policy" annotation)?

### ISSUE THREAD

**Issue by kastl-ars** [helm/helm#13202]
Hi,

sorry if this is documented somewhere, all my search foo failed me. So for the next one looking for a solution:

The helm docs previously recommended (or falsely required) quotes around annotations like `helm.sh/resource-policy`.

```
metadata:
  annotations:
    "helm.sh/resource-policy": keep
```

- https://github.com/helm/helm-www/pull/1552
- https://github.com/helm/helm/issues/12555

So those are pretty common in the wild.

**What is the right way to overwrite or remove such an annotation (by setting it to `null`)?**

- Quoting the key in the overrides does create an additional annotation **without** quotes
- Using `'"helm.sh/resource-policy"` creates the key with single and double quotes
- Using `"\"helm.sh/resource-policy\""` creates an annotation with the backslashes

Puzzled,
Johannes

**Comment by kastl-ars** [comment]
OK, I have once again been bitten by Helm not being able to remove values in subcharts (or dependencies) by setting `null` in the values.yaml.

If I trim down my example I can overwrite the annotation by just using the same quoted way in the values.yaml.

TL;DR: Just use the annotation as they are, with quotes.
